dask/bag/tests/test_text.py

```diff
@@ -48,9 +53,20 @@ def test_read_text(fmt, bs, encoding):
         (L,) = compute(b)
         assert "".join(L) == expected
 
-        b = read_text(sorted(files), compression=fmt, blocksize=bs, encoding=encoding)
+        o = read_text(
+            sorted(files),
+            compression=fmt,
+            blocksize=bs,
+            encoding=encoding,
+            include_path=include_path,
+        )
+        b = o.pluck(0) if include_path else o
         (L,) = compute(b)
         assert "".join(L) == expected
+        if include_path:
+            (paths,) = compute(o.pluck(1))
```

Comment:
Can we be a bit stronger about this assertion? Some kind of assertion that all of the lines within a partition have the expected path?